ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -1,6 +1,7 @@
 import QtQuick 2.15
 import QtQuick.Controls 2.15
 import QtQuick.Layouts 1.15
+import QtQuick.Controls.Universal 2.15
```

Comment:
I think this is unnecessary.

Reply:
It is necessary for the tablet, I added it on Alex's request and we tested it together

Reply:
This also looks like a bit shady workaround. Probably calling `QQuickStyle::setStyle(QStringLiteral("Universal"));` is remedy for that as well.